script.js

```diff
@@ -1,4 +1,1250 @@
+var Code01 = "PROJECT PART 1: SCISSORS PAPER STONE: PART 1-STANDARD MODE";
+
+// random number generator
+
+/* var randomNumber = function () {
+  var randomDecimal = Math.random() * 3;
+  var randomInteger = Math.floor(randomDecimal);
+  return randomInteger + 1;
+};
+
+// read random Number to output  random GameHand
+var readGameHandNmbr = function (gameHandNmbr) {
+  var gameHandPlayed = "";
+  if (gameHandNmbr == 1) {
+    gameHandPlayed = "scissors";
+  }
+  if (gameHandNmbr == 2) {
+    gameHandPlayed = "paper";
+  }
+  if (gameHandNmbr == 3) {
+    gameHandPlayed = "stone";
+  }
+  return gameHandPlayed;
+};
+
+var main = function (input) {
+  var gameHandRandom = randomNumber();
+  var gameHand = readGameHandNmbr(gameHandRandom);
+
+  var draw = `The computer chose ${gameHand.toUpperCase()}.<br><br>You chose ${input.toUpperCase()}.<br><br> =It's a DRAW!=`;
+  var lose = `The computer chose ${gameHand.toUpperCase()}.<br><br>You chose ${input.toUpperCase()}.<br><br> ~You LOSE~`;
+  var win = `The computer chose ${gameHand.toUpperCase()}.<br><br>You chose ${input.toUpperCase()}.<br><br> ***You WIN!***`;
+  var errorMessage = `xxxxxxxxx INPUT ERROR xxxxxxxxxx. <br><br> There are only 3 options: 'scissors', 'paper' or 'stone'.`;
+  var endingMessage = `<br><br> To play the next game, please type 'scissors', 'paper' or 'stone'.`;
+
+  var myOutputValue = errorMessage;
+
+  var judgement = function (playerHand, computerHand) {
+    if (playerHand == computerHand) {
+      return "draw";
+    }
+    if (playerHand == "scissors" && computerHand == "stone") {
+      return "lose";
+    }
+    if (playerHand == "scissors" && computerHand == "paper") {
+      return "win";
+    }
+    if (playerHand == "paper" && computerHand == "scissors") {
+      return "lose";
+    }
+    if (playerHand == "paper" && computerHand == "stone") {
+      return "win";
+    }
+    if (playerHand == "stone" && computerHand == "paper") {
+      return "lose";
+    }
+    if (playerHand == "stone" && computerHand == "scissors") {
+      return "win";
+    }
+  };
```

Comment:
You can combine the conditions that output the same outcome into a single if statement using the || operator. for example:

`if (playerHand == "scissors" && computerHand == "paper") ||
(playerHand == "paper" && computerHand == "stone") ||
(playerHand == "stone" && computerHand == "scissors") {
return "win";
}`